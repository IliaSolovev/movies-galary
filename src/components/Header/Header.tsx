import React from 'react';

import style from './Header.module.scss';

export const Header:React.FC = (props) => (
  <header className={style.header}>
    {props.children}
  </header>
);
