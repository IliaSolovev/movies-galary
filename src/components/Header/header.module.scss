@import '../../styles/variables';
@import '../../styles/main';
.header{
  position: relative;
  width: 100vw;
  height: 20vh;
  background: url("./images/headerBg.jpg") 50% 50% no-repeat;
  padding: 15px 0 30px 0;
  z-index: 1;
  &:before{
    content: "";
    position: absolute;
    opacity: .3;
    background: #000000;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    z-index: 2;
  }
  & > * {
    @extend .headerContainer;
    position: relative;
    z-index: 3;
  }
}
.header__title{
  margin-bottom: 2vw;
  font-family: "Roboto",sans-serif;
  font-size: 1.2em;
  font-weight: bold;
  color: $redColor;
}
