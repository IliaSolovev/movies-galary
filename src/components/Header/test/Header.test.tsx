import React from "react";
import {shallow} from "enzyme";
import Header from "../Header";
it('should render header', () => {
    const component = shallow(<Header/>);
    expect(component).toMatchSnapshot();
});