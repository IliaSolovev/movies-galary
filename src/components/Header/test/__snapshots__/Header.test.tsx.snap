// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<header
  className="header"
/>
`;
